meganav.yaml

```diff
@@ -34,5 +724,175 @@ get_ubuntu:
+            - title: MAAS
+              description: Bare metal provisioning
+              url: https://maas.io/
+            - title: Certified hardware
+              description: Machines you can trust
+              url: /certified/servers
+            - title: Public clouds
+              description: Server on AWS, Azure and GCP
+              url: /cloud/public-cloud
+            - title: Multipass
+              description: VMs for Windows, macOS & Linux
+              url: https://multipass.run/
+      secondary_links:
+        - title: Tutorials and docs
+          links:
+            - title: Server guide
+              url: /server/docs
+            - title: Installation tutorial
+              url: /tutorials/install-ubuntu-server#1-overview
+            - title: ARM
+              url: /download/server/arm
+            - title: IBM Power
+              url: /download/server/power
+            - title: s390x
+              url: /download/server/s390x
+            - title: RISC-V
+              url: /download/risc-v
+      section_footer:
+        cta_title: Get Ubuntu Server
+        cta_url: /download/server
+    - title: Raspberry Pi
+      primary_links:
+        - title: Products
+          links:
+            - title: Ubuntu for Raspberry Pi
+              description: Single board computer
+              url: /raspberry-pi
+            - title: Install on Raspberry Pi
+              description: Download per-model
+              url: /download/raspberry-pi
+            - title: Ubuntu Core for Raspberry Pi
+              description: Embedded OS
+              url: /download/raspberry-pi-core
+            - title: Ubuntu-ready Pi devices
+              description: Certified hardware
+              url: /certified?q=&limit=20&vendor=Raspberry+Pi+Foundation
+      secondary_links:
+        - title: Tutorials and docs
+          links:
+            - title: Desktop installation tutorial
+              url: /tutorials/how-to-install-ubuntu-on-your-raspberry-pi#1-overview
+            - title: Server installation tutorial
+              url: /tutorials/how-to-install-ubuntu-on-your-raspberry-pi#1-overview
+            - title: Core install instructions
+              url: /download/raspberry-pi-core
+            - title: Introduction video
+              url: /engage/raspberry-pi-livestream
+            - title: Build a Kubernetes cluster on Raspberry Pi
+              url: /tutorials/how-to-kubernetes-cluster-on-raspberry-pi#1-overview
+      section_footer:
+        cta_title: Get Ubuntu for Raspberry Pi
+        cta_url: /download/raspberry-pi
+    - title: Ubuntu Core for IoT
+      primary_links:
+        - title: Products
+          links:
+            - title: Ubuntu Core
+              description: OS designed for IoT
+              url: /core
+            - title: Internet of Things
+              description: Ubuntu for IoT
+              url: /internet-of-things
+            - title: Smart things as a service
```

Comment:
```suggestion
            - title: Smart Start
```

Reply:
Smart Start as a brand name was retired, I'll update the spreadsheet